inlong-agent/conf/agent.properties

```diff
@@ -101,3 +101,10 @@ channel.memory.capacity=5000
 agent.manager.vip.http.host=
 agent.manager.vip.http.port=
 
+
+
+############################
+# prometheus config
+############################
+# prometheus exporter server default port
+agent.prometheus.exporter.port=8080
```

Comment:
It is recommended to use some infrequently used ports.

Reply:
@shink it's better to add a config to enable or disable expose Prometheus HTTP server.